src/MigrationTools.Clients.AzureDevops.ObjectModel/ProcessorEnrichers/TfsNodeStructure.cs

```diff
@@ -283,7 +284,19 @@ private string NodeStructureTypeToLanguageSpecificName(TfsLanguageMapOptions lan
         private void ProcessCommonStructure(string treeTypeSource, string sourceTarget, string treeTypeTarget, string projectTarget)
         {
             Log.LogDebug("NodeStructureEnricher.ProcessCommonStructure({treeTypeSource}, {treeTypeTarget})", treeTypeSource, treeTypeTarget);
-            NodeInfo sourceNode = (from n in _sourceRootNodes where n.Path.Contains(treeTypeSource) select n).Single();
```

Comment:
Was there a reason you were doing `.Contains`? I'm concerned there's some legacy TFS server that doesn't include the project name in the node.Path, and this will break all of them. It would be good to have someone more knowledgeable about the project verify that my fix will still work for all TFS servers.

Reply:
All versions of TFS have the project name in the node path.